apps/sanity/schemas/faqs.ts

```diff
@@ -0,0 +1,40 @@
+import { defineType, defineField, defineArrayMember } from "sanity";
+
+export default defineType({
+	name: "faqs",
+	title: "FAQs",
+	type: "document",
+	fields: [
+		defineField({
+			name: "faqs",
+			title: "FAQs",
+			type: "array",
+			of: [
+				defineArrayMember({
+					type: "object",
+					name: "faq",
+					fields: [
+						defineField({
+							name: "question",
+							title: "Question",
+							type: "text",
+						}),
+
+						defineField({
+							name: "answer",
+							title: "Answer",
+							type: "text",
+						}),
+					],
+
+					preview: {
+						select: {
+							title: "question",
+							subtitle: "answer",
+						},
+					},
```

Comment:
Proposal: Instead of using the default icon, would it be better to use another? (Maybe [badge help](https://lucide.dev/icons/badge-help) for the sidebar and [file question](https://lucide.dev/icons/file-question) for each question?)